lib/src/base-styles/_nx-tabs.scss

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+ /*
+  #Tabs
+*/
+.nx-tabs {
```

Comment:
There is no reason for any of the classes below to be nested under `.nx-tabs` they should all be stand alone classes.

Reply:
a41dd05